conda_windows.md

```diff
@@ -0,0 +1,88 @@
+---
+ Getting Going with Python on a Windows
```

Comment:
Need to put `Title:` here in between the `---` lines in order for jekyll to find this properly (this caused the test error)